Services/DataX.Utilities.JobRunner/DataX.Utilities.JobRunner/StorageTableBase.cs

```diff
@@ -0,0 +1,81 @@
+﻿// *********************************************************************

```

Comment:
this is also a dup of Services/DataX.Utilities.JobRunner/StorageTableBase.cs

Reply:
Updated